src/main/java/com/example/gradecalculator/entities/AllUsers.java

```diff
@@ -0,0 +1,57 @@
+package com.example.gradecalculator.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class AllUsers {
```

Comment:
needs to be renamed to User

Reply:
in last two branches its User, "Users Data should be now saved in database " was done before these branches